test/Templates.Test/CdnScriptTagTests.cs

```diff
@@ -70,6 +88,21 @@ public async Task CheckSubresourceIntegrity(ScriptTag scriptTag)
             Assert.Equal(expectedIntegrity, scriptTag.Integrity);
         }
 
+        [Theory]
+        [MemberData(nameof(SubresourceIntegrityCheckLinkData))]
+        public async Task CheckLinkSubresourceIntegrity(LinkTag linkTag)
+        {
+            string expectedIntegrity;
+            using (var responseStream = await _httpClient.GetStreamAsync(linkTag.HRef))
+            using (var alg = SHA256.Create())
+            {
+                var hash = alg.ComputeHash(responseStream);
+                expectedIntegrity = "sha256-" + Convert.ToBase64String(hash);
+            }
+
+            Assert.Equal(expectedIntegrity, linkTag.Integrity);
```

Comment:
Glad we have a test for this! Can this test also be used to help update the integrity value if the scripts are updated? If so, maybe it makes sense to add a nice comment to this test (or others??) that says something like: "If this test has failed, but the script/CSS/whatever files changed on purpose, use the new integrity values in the 'content' files."

Reply:
I don't like suggesting the fix because it could have been broken in multiple ways (accidentally changed the file from what the "content" says, referencing the wrong content, etc), but what if we just give a more descriptive error like: `Expected {linkTag.HRef} to have Integrity '{expectedIntegrity}' but it had '{linkTag.Integrity}'.`

Reply:
OK